docs/advanced/global_configuration.md

```diff
@@ -0,0 +1,29 @@
+# Global configuration in pixi
+
+Pixi supports some global configuration options, as well as project-scoped configuration (that does not belong into the project file).
+The configuration is loaded in the following order:
+
+1. Global configuration folder (e.g. `~/.config/pixi/config.toml` on Linux, dependent on XDG_CONFIG_HOME)
+2. Global .pixi folder: `~/.pixi/config.toml` (or `$PIXI_HOME/config.toml` if the `PIXI_HOME` environment variable is set)
```

Comment:
Is this required? I would like to have only one global config.

Reply:
There is also a use case where system administrators want to put a global config into a folder that is not user-editable (e.g. to forcibly configure the channels that should be used).

We already have the logic to _merge_ configurations, so I think it doesn't really complicate much to allow multiple locations for loading. 

We should implement a `pixi config` subcommand later on which allows for `pixi config set --global bla true`. This would choose a default location to edit the toml file.